ui/src/dashboards/containers/DashboardPage.tsx

```diff
@@ -138,6 +139,8 @@ class DashboardPage extends Component<Props, State> {
     const annotationRange = millisecondTimeRange(timeRange)
     getAnnotationsAsync(source.links.annotations, annotationRange)
 
+    AutoRefresh.poll(autoRefresh)
```

Comment:
Passing `0` to `AutoRefresh.poll` as a way of saying “dont poll at all” is a bit confusing to me. I feel like it should just not be called

Reply:
maybe a method called `stopPolling` would be clearer.